src/posix/platform/include/openthread/openthread-system.h

```diff
@@ -254,6 +254,17 @@ void otSysSetInfraNetif(const char *aInfraNetifName, int aIcmp6Socket);
  */
 bool otSysInfraIfIsRunning(void);
 
+/**
+ * Initializes the CLI module using the daemon.
+ *
+ * This function initializes the CLI module, and assigns the daemon to handle
```

Comment:
Better document:
1. the typical use case
2. this method can be called for multiple times